packages/demand-bloc-track-trip/src/TripBloc.ts

```diff
@@ -5,16 +5,21 @@ import {
   Fare,
   FinalFareResponse,
   CancellationParams,
+  BookATripResponse,
+  SearchParams,
 } from '@karhoo/demand-api'
 import { Subject, Subscription } from 'rxjs'
 import polling from './polling'
 
 import { publishReplay, refCount } from 'rxjs/operators'
-import { FinalTripStatuses, FinalFareStatuses } from './statuses'
+import { FinalTripStatuses, FinalFareStatuses, TripStatuses } from './statuses'
 import { tripTransformer, TripFollowResponse } from './tripTransformer'
+import { makeSearchParams } from './utils'
 
 const POLLING_INTERVAL_TRACK = 10000
 const POLLING_FINAL_FARE = 20000
+const defaultUpcomingTripsRow = 2
+const defaultPastTripsRow = 10
```

Comment:
I'd suggest to stick to the same code styling for constants, either `THIS_STYLE` or `thatStyle`